step2-02/README.md

```diff
@@ -65,9 +65,9 @@ const MyComponent = () => {
 };
 ```
 
-## Render Props
+### Customizing Component Rendering
 
-Some Fabric components take in a render function like the TextField:
+Some Fabric components take in a render functions to allow customizing certain parts of the component. An example with TextField:
```

Comment:
Might be nice if we could come up with a slightly less contrived example (some simple real-world case where you'd want to customize rendering)

Reply:
yeah would be nice... i can't think of any right now...

Reply:
me either :(